application-ideas-ui/src/main/resources/Ideas/RecentIdeasMacro.xml

```diff
@@ -251,7 +251,7 @@
         #end
         [[$document.title&gt;&gt;$idea]] $services.localization.render('ideas.author', ["[[$xwiki.getUserName($document.author, false)&gt;&gt;$document.author]]"]) $services.localization.render('ideas.date', [$xwiki.formatDate($document.date, 'dd/MM/yyyy hh:mm')])
         &lt;span class="hidden status"&gt;$document.status&lt;/span&gt;
-        &lt;span class="hidden fullName"&gt;$document.fullName&lt;/span&gt;
+        &lt;span class="hidden fullName"&gt;$services.ideas.url($document.documentReference)&lt;/span&gt;
```

Comment:
See my comment above. You should compute the REST URL in JavaScript. In any case, this is not the right way to pass additional information from the server to the client. Best is to use ``data-`` attributes (e.g. on the idea DIV above).